modules/ROOT/pages/managing-users.adoc

```diff
@@ -24,11 +24,11 @@ Configuring SAML for SSO involves:
 
 == Configure SAML in Identity Management
 
-. In Anypoint Platform, sign into the master organization as a user with the *Organization Administrators*.
+. In Anypoint Platform, sign into the master organization as a user with the *Organization Administrator* permission.
```

Comment:
The verbs are to sign in (to) and to log in (to)
```suggestion
. In Anypoint Platform, sign in to the master organization as a user with the *Organization Administrator* permission.
```
Is Org Admin a UI label? Otherwise I would not format it in bold.

Reply:
```suggestion
. In Anypoint Platform, sign in to the master organization as a user with the Organization Administrator permission.
```